class/defaults.yml

```diff
@@ -12,3 +12,7 @@ parameters:
         http01:
           ingress:
             class: 'nginx'
+    images:
+      kubectl:
+        image: docker.io/bitnami/kubectl
```

Comment:
FYI: We do have a oc image that we use for Openshift: https://github.com/appuio/container-oc

Reply:
btw, we should avoid docker.io whenever possible due to the Pull limit...

Reply:
ack 👍

`bitnami/kubectl` is on quay too. Which image do you prefer? `bitnami/kubectl` "feels" a bit more generic to me, but I don't mind at all using `oc`.

Reply:
I have no hard feeling here. It just has to work for both plain K8s and Openshift.